src/ServiceInsight/Framework/Licensing/AppLicenseManager.cs

```diff
@@ -56,4 +45,20 @@ public int GetRemainingTrialDays()
 
         public bool IsLicenseExpired() => LicenseExpirationChecker.HasLicenseExpired(CurrentLicense);
     }
+
+    internal class InMemoryLicenseSource : LicenseSource
```

Comment:
Where is this used? I can't see where this is added to the result returned by `GetStandardLicenseSources`

Reply:
Removed. I had refactored to use `LicenseDialogSource` instead.